webpay/pay/views.py

```diff
@@ -157,7 +158,12 @@ def configure_transaction(request, data=None):
     else:
         sim = (request.session['notes']['pay_request']['request']['simulate']
                if is_simulation else None)
-        return {'status': 'ok', 'simulation': sim}
+        client_trans_id = 'client-trans:{u}'.format(u=uuid.uuid4())
```

Comment:
I'm wondering if the difference between the trans_id in the session and this reference needs to be clearer to avoid future confusion/conflation?